qemu/tests/plug_cdrom.py

```diff
@@ -0,0 +1,120 @@
+import ast
+import logging
+import re
+
+from virttest import error_context
+from virttest import utils_test
+from virttest import utils_misc
+
+from provider.block_devices_plug import BlockDevicesPlug
+
+HOTPLUG, UNPLUG = ('hotplug', 'unplug')
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test plug cdrom device.
+      Scenario "with_hotplug":
+        1) Start VM with virtio-scsi-pci (system disk).
+        2) For windows, Check whether vioscsi.sys verifier enabled in guest.
+        3) Hot plug a virtio-scsi cdrom via qmp.
+        4) Check in qmp monitor.
+        5) Check in VM, the cdrom show in system computer.
+        6) For linux, check guest kernel logs.
+        7) Reboot or shutdown guest.
+
+      Scenario "with_unplug":
+        1) Boot a guest with virtio-scsi CD-ROM normally.
+        2) Check block in QMP.
+        3) Delete the virtio-scsi CD-ROM by qmp.
+        7) Reboot or shutdown guest.
```

Comment:
This number is wrong,
 and it is best to check guest kernel logs too for linux after with_unplug. because the bug often appear in unplug.

Reply:
> This number is wrong,
Thanks, will modify the wrong line number.
> and it is best to check guest kernel logs too for linux after with_unplug. because the bug often appear in unplug.
There are not such steps to check kernel logs, pls add this steps if you need it.
cc @XueqiangWei  @peixiu